.github/workflows/medic-benchmarks.yml

```diff
@@ -27,12 +27,15 @@ jobs:
           echo "normal=$(kubectl get ns -o name | grep -E namespace/medic-cw-[0-9]+-[a-z0-9]+-benchmark$ | sort -V | head -n1)" >> $GITHUB_OUTPUT
       - name: Delete benchmarks
         run: |
-          kubectl delete ${{ steps.data.normal }}
-          kubectl delete ${{ steps.data.normal }}-mixed
-          kubectl delete ${{ steps.data.normal }}-latency
+          kubectl delete ${{ steps.data.outputs.normal }}
+          kubectl delete ${{ steps.data.outputs.normal }}-mixed
+          kubectl delete ${{ steps.data.outputs.normal }}-latency
   benchmark-data:
     name: Collect benchmark data
     runs-on: ubuntu-latest
+    outputs:
+      ref: ${{ steps.data.outputs.ref }}
```

Comment:
:x: There seems to be an issue with the `ref`. 

See https://github.com/camunda/zeebe/actions/runs/3618326752/jobs/6098054983#step:2:6
It passed an empty `ref` to the benchmark creation: https://github.com/camunda/zeebe/actions/runs/3618326752/jobs/6098090182

@oleschoenburg please have a look.